### DISCUSSION TITLE
Snap Package Available!

### DISCUSSION THREAD

**Discussion by Jeffser** [Jeffser/Alpaca#329] [Announcements]
Hi everyone, after some help from [hideo aoyama](https://github.com/boukendesho) and [সৌম্যদীপ ঘোষ ](https://github.com/soumyaDghosh), and some testing on a virtual machine, I'm happy to announce that Alpaca now has a Snap package!

To install it please download the Snap from the [latest release](https://github.com/Jeffser/Alpaca/releases) and install it using this command:
```BASH
sudo snap install ./{package name} --dangerous
```
The `--dangerous` comes from the package being installed without any involvement of the SnapStore, I'm working on getting the app there, but for now you can test the app this way.

The instructions have also been added to the [README](https://github.com/Jeffser/Alpaca/blob/main/README.md)

I also submitted a request for the name `alpaca` or `jeffser-alpaca` on the SnapStore, once I have the name I will publish the app there.

Thanks for all the help and support, please report any problems related with the Snap package!

**Comment by olumolu** [comment]
Will you push snap on 3.0.0

**Reply by Jeffser** [reply]
I'll try to get it all fixed up, if everything goes well I might actually try to get it published in Ubuntu's snap store

**Reply by Jeffser** [reply]
I fixed the snap version of 2.9.0, I tried to publish it to the snap store but I keep getting some errors, I made an issue in the snapcraft repo, hopefully I can get this sorted out before 3.0.0
